src/routes/page-routes/issues.tsx

```
@@ -0,0 +1,39 @@
+import Issues from '~/components/Issues/Issues';
+import { ISSUES_PATH } from '../paths';
+import { RouteErrorBoundry } from '../RouteErrorBoundary';
+import IssuesListPage from '~/components/Issues/IssuesListPage';
```

ESLint

[error] `~/components/Issues/IssuesListPage` import should occur before import of `../paths`

(import/order)